app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/coverage/CoveredAndExceptionModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.hedvig.app.feature.insurance.ui.detail.coverage
+
+sealed class CoveredAndExceptionModel {
```

Comment:
Now that the adapter is called `PerilAdapter`, we might want to rename this to something more appropriate 🙂